### DISCUSSION TITLE
Wie funktioniert die Drosselung im Multi Inverter Modus

### DISCUSSION THREAD

**Discussion by programmdesign** [tbnobody/OpenDTU#1498] [Q&A]
Hallo. Ich würde gerne 2 Hoymiles WR zusammenschalten und dann das Gesamtsystem auf 600W drosseln. Ist dies gesamthaft möglich -- also stelle ich an einer Stelle 600W ein oder muss ich z.B. je 300W je für einen Wechselrichter einstellen? Leider konnte ich dazu keine Infos oder Doku finden. Schonmal vielen Dank!

**Comment by DPO99** [comment]
Was genau ist denn der Multi Inverter Modus? Das gibt es m.E. nicht. Somit muss jeder WR einzeln eingestellt werden. Je nachdem, wie die ausgerichtet sind, sind im Zweifel 300+300 keine 600.

**Reply by Archibald-Harry-Tuttle** [reply]
Wie @DPO99  schon schrieb: multi inverter mode gibt es nicht.
Die Hoymiles lasen sich ab Werk untereinander in Serie schalten was aber kaum mit dt. VDE, Balkonsolar etc.  vereinbar sein dürfte.
Leitungen beider Hoymiles würden sich auf eine 230V Anschlussdose aufsummieren.
Also besser jeden Hoymiles an eine Steckdose, unterschiedliche Phasen wären nicht verkehrt.
Verwende je einen Shelly Plug S (oder ähnliche) vor den Hoymiles und mache entweder Ost-/ West-Hoymiles zeitgesteuert mit je 600Watt.
Also wenn der Eine produziert, ist der andere aus.
Oder etwas mit Homeautomation (Mqtt) und diversen Regeln die Du setzen kannst.
Bsp.: wenn Ost weniger als 600W produziert, dann schalte West mit max. der Differenz zu den 600w hinzu.

**Reply by programmdesign** [reply]
Danke für die Antworten. @DPO99 "es was Multi Inverter Support" gemeint (siehe unter https://github.com/tbnobody/OpenDTU#features-for-end-users in der Feature-Liste). Das heißt vermutlich nur, dass ich mit der OpenDTU mehrere Inverter gleichzeitig ansprechen kann, richtig?

**Reply by fkoehr** [reply]
Ja, das ist richtig. Aber es ist durchaus sinnvoll, für jeden Wechselrichter einen eigenen Aufbau zu verwenden. Je nach Ausstattung des Micocontrollers ... aber die Redundanz dezentraler Systeme ist unschlagbar ;o)
